### DISCUSSION TITLE
how do I open "settings"?

### DISCUSSION THREAD

**Discussion by citydelver** [zoidsh/meru#241] [Q&A] [answered]
Hello. Sorry for the probably obvious question, but how do I get to change settings? Can't find them anywhere...

**Comment by timche** [comment, marked as answer]
Which OS are you using? 

- If you are on macOS, the settings can be found in the system menu bar on the top left of your screen while the app is focused.
- If you are on Linux or Windows, it can be found in the app menu bar on top of the app.
	- If the app menu bar is not visible, then it might be hidden and it can be toggled with <key>Alt</key>.
